src/Symfony/Component/Workflow/MarkingStore/MultipleStateMarkingStore.php

```diff
@@ -22,6 +22,8 @@
  * This store deals with a "multiple state" Marking. It means a subject can be
  * in many states at the same time.
  *
+ * @deprecated since Symfony 4.3. Use MethodMarkingStore instead.
```

Comment:
should trigger a runtime notice also, below the namespace (same in SingleStateMarkingStore)

Reply:
Thanks. Fixed.